src/evaluation/Run.jl

```diff
@@ -181,21 +181,26 @@ function update_save_run!(session::ServerSession, notebook::Notebook, cells::Arr
 	end
 
 	if !(isempty(to_run_online) && session.options.evaluation.lazy_workspace_creation) && will_run_code(notebook)
-		run_task = @async begin
-			run_reactive!(session, notebook, old, new, to_run_online; kwargs...)
-		end
-		if run_async
-			run_task
-		else
-			fetch(run_task)
-		end
+		maybe_run_async!(session, notebook, old, new, to_run_online; run_async=run_async, kwargs...)
 	end
 end
 
 update_save_run!(session::ServerSession, notebook::Notebook, cell::Cell; kwargs...) = update_save_run!(session, notebook, [cell]; kwargs...)
 update_run!(args...) = update_save_run!(args...; save=false)
 
 
+maybe_run_async!(session::ServerSession, notebook::Notebook, to_run::Vector{Cell}; kwargs...) =
```

Comment:
Can we change this to `run_maybe_async!`?

Reply:
:heavy_check_mark: Renamed and moved just after `run_reactive!`